tsconfig.build.json

```diff
@@ -12,7 +12,8 @@
     "noImplicitThis": false,
     "esModuleInterop": true,
     "importHelpers": true,
-    "noEmitHelpers": false
+    "noEmitHelpers": false,
+    "useUnknownInCatchVariables": false
```

Comment:
there's a bunch of (admittedly correct) errors about not narrowing `error` in `catch` blocks -- going to leave those for another day